plugins/item_tasks/web_client/views/SelectSingleFileTaskWidget.js

```diff
@@ -59,7 +59,7 @@ var SelectSingleFileTaskWidget = View.extend({
         this.task = params.task;
         this.$('.g-submit-select-task').girderEnable(true);
 
-        this.$('.g-target-result-container').html(selectTaskViewDescriptionTemplate({
+        this.$('.g-target-result-container').html(SelectSingleFileTaskWidgetSelectedTemplate({
```

Comment:
I agree that it adds seemingly unnecessary complexity, but I think it is easy to read and allows easier design/arrangement changes down the line.